app/views/shared/_landingfooter.erb

```diff
@@ -0,0 +1,81 @@
+    <!-- Footer -->
```

Comment:
I don't understand why this isn't a `.html.erb` file.  I suspect Rails will render it as HTML, but ???

Reply:
fixed.